config/profile/rhdh/default-config/pvcs.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: audit-log
+  annotations:
+    "rhdh.redhat.com/mount-path": "/var/log/redhat-developer-hub/audit"
+spec:
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Filesystem
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: dynamic-plugins
```

Comment:
Should be mounted at `/opt/app-root/src/dynamic-plugins-root` in the main RHDH container and at `/dynamic-plugins-root` in initContainer

Reply:
Ok, so the goal is to replace what we have as a part of deployment...
Removing this placeholder and opened the issue for dynamic plugins: https://issues.redhat.com/browse/RHIDP-4595